server/routes/api.js

```diff
@@ -0,0 +1,11 @@
+const Router = require('express').Router;
+const db = require('../db');
+
+const app = new Router();
+
+app.get('/', (req, res) => {
+    console.log(db);
```

Comment:
This will log the db object to the console when hitting `/api` exactly, so I could see it. Gotta remove it when you start adding business logic.